entity-framework/core/modeling/generated-properties.md

```diff
@@ -48,7 +49,7 @@ Like `value generated on add`, if you specify a value for the property on a newl
 
 ## Conventions
 
-By convention, non-composite primary keys of type short, int, long, or Guid will be setup to have values generated on add. All other properties will be setup with no value generation.
+By default, non-composite primary keys of type short, int, long, or Guid will be setup to have values generated on add. All other properties will be setup with no value generation.
```

Comment:
Is it still true that you don't get value generation if the property is part of a composite? I vaguely remember that we (possibly unintentionally) changed this.

Reply:
Yes, we changed this back for non-owned types. And I don't think we need to talk about owned types here.